gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -196,6 +196,7 @@ def table_exist(client, project_id, dataset_id, table_id):
       raise
   return True
 
+
```

Comment:
nit: unnecessary line.

Reply:
There are two empty lines between all functions in this module, I just made this one consistent with the rest of them.